app/build/outputs/logs/manifest-merger-debug-report.txt

```diff
@@ -1,140 +1,140 @@
 -- Merging decision tree log ---
```

Comment:
repository should contains only source code, without build files and debug-reports. These files should be removed.